helm/macropower-analytics-panel-server/templates/ingress-auth.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
Ingress v1 is not supported in some installations. I am fine with us using it but we need to feature flag it like it https://github.com/goharbor/harbor-helm/blob/37dc4069bb0b0e03bfa79b4a915d479267a2d247/templates/ingress/ingress.yaml#L31

Reply:
All good I'll revert this.